Mage.Client/src/main/java/org/mage/plugins/card/utils/CardImageUtils.java

```diff
@@ -218,6 +218,7 @@ public static String buildImagePathToCard(CardDownloadData card) {
         if (cardName.isEmpty()) {
             cardName = prepareCardNameForFile(card.getName());
         }
+        cardName = cardName.replaceAll("[\\\\/:*?\"<>|]", "");
```

Comment:
It's already have `prepareCardNameForFile` before that line. If you got some bad symbol then add it to that procedure.

Reply:
I removed that line from the file and recommitted, but either that change caused the Travis CI build to fail, or the check itself is inconsistent (since the previous build passed).